copulas/visualization.py

```diff
@@ -111,7 +111,7 @@ def compare_1d(real, synth, columns=None, figsize=None):
     fig = plt.figure(figsize=figsize)
 
     for idx, column in enumerate(columns):
-        position = prefix + str(idx + 1)
+        position = int(prefix + str(idx + 1))
```

Comment:
This change is needed to avoid the following error: `ValueError: Single argument to subplot must be a three-digit integer, not '211'`. This seems unrelated to my code changes, and due to an update in the API of matplotlib.